### ISSUE TITLE
`{field.emailAddress}` outputs value surrounded by `p` tags in Recipients field.

### ISSUE THREAD

**Issue by cavellblood** [verbb/formie#515]
**Description**
A notification email is supposed to be sent the the user's email but I'm getting this error after submitting a form: 

```
Failed to send notification email: {"error":"Notification email parse error for “To: {field.emailAddress}”. Template error: “Address in mailbox given [<p>user@site.com</p>] does not comply with RFC 2822, 3.6.2.”
```

It looks like it's adding a paragraph tag around the email address and it seems that is what is causing the error. I thought it might be just an issue with my form and it's input but I created a new Contact form from the provided stencil and it behaved the same way. Buy using the stencil from Formie that also proved that the custom form templates I'm using (which are only very slightly modified and only for checkboxes, not email/text fields) are not the cause of the issue because the stencil uses the the Default Formie Templates.

The other closest issues that I could find that might be able to help me were #217, #284, and #290. It was mentioned that version `1.3.8` of Formie corrected this issue.

**Steps to reproduce**
1. Add new form based off of Contact Form stencil. But it seems to be have the same way with any email address field on any form.
2. Render the form on the front end with Default Formie templates.
3. Submit the form

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: Yes **and** No (Error happens with and without custom form templates.)

**Additional info**
- Plugin version: `1.4.8`
- Craft version: `3.6.17`
- Multi-site: Yes

**Additional context**
Error in Queue Manager:
<img width="1469" alt="Screen Shot 2021-07-05 at 4 44 57 PM" src="https://user-images.githubusercontent.com/727740/124519923-77462380-ddb0-11eb-91f3-4dbc38580a6e.png">

How the User notification is setup:
<img width="1395" alt="Screen Shot 2021-07-05 at 4 50 38 PM" src="https://user-images.githubusercontent.com/727740/124520209-5b8f4d00-ddb1-11eb-8974-53b77dfd2041.png">

How the form is rendered on the front end:
<img width="423" alt="Screen Shot 2021-07-05 at 4 53 48 PM" src="https://user-images.githubusercontent.com/727740/124520295-9beecb00-ddb1-11eb-8f19-f262f3823c49.png">

**Comment by cavellblood** [comment]
I just downgraded Formie to `1.4.6` and it works as expected.

**Comment by janhenckens** [comment]
We're also running into this issue on `1.4.8`.

**Comment by jasondibenedetto** [comment]
Can confirm we are also getting this error on `1.4.8`

**Comment by AndrewMac** [comment]
Yep, add one more person to the list of those getting this new error!

**Comment by engram-design** [comment]
Thanks for all the "+1s" although I can't replicate this on existing or fresh installs on `1.4.8`.

Just to confirm the `emailAddress` field is an Email Address type field?

You mentioned creating a new form from a stencil. Does the same issue happen when creating it from a blank form (so no stencil)?

**Comment by engram-design** [comment]
Ah, managed to figure this out. Fixed for the next release, which will be out shortly. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.8",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.9](https://github.com/verbb/formie/releases/tag/1.4.9)
